.travis.yml

```diff
@@ -1,16 +1,22 @@
 language: python
 python:
 - 2.7
-- 3.6
+- 3.5
 env:
 - TOXENV=django111
 - TOXENV=django20
+- TOXENV=django21
```

Comment:
where are TOXENV defined?

Reply:
I have added this question on other ticket also.

Reply:
The Travis configuration here doesn't really do anything except deploy to PyPI on tags (the test script is `echo "Success"`).  There's no tox configuration, and not even any unit tests.  So there isn't much point in testing multiple Python or Django versions here, there just aren't any test cases for them to make a difference on.